python/mlc_chat/cli/chat.py

```diff
@@ -0,0 +1,74 @@
+"""A command line tool for directly chatting with open source LLMs"""
+from mlc_chat import ChatModule
+from mlc_chat.callback import StreamToStdout
+import os
+from ..support.argparse import ArgumentParser
+
+
+
+# From the mlc-llm directory, run
+# $ python mlc_chat_cli.py
+
+def printhelp():
```

Comment:
`ArgumentParser` actually auto generates the help message. Use `parser.print_help()`

Reply:
Thanks.   Method renamed.  Actually it prints all available internal slash commands like  `/reset`.